mitodl/ocw-studio
# Integrate Touchstone login

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #62 

#### What's this PR do?
This adds integration with Touchstone to enable user account creation/login.

#### How should this be manually tested?
Follow the instructions in the README for setting up your local environment w/ SSOCircle and verify you can login w/ it.
